openpype/tools/publisher/window.py

```diff
@@ -421,6 +431,21 @@ def keyPressEvent(self, event):
         if event.key() == QtCore.Qt.Key_Escape:
             event.accept()
             return
+
+        if event.matches(QtGui.QKeySequence.Save):
+            self._save_changes(True)
+            event.accept()
+            return
+
+        if (
+            event.modifiers() == QtCore.Qt.ControlModifier
+            and event.key() == QtCore.Qt.Key_R
+        ):
+            if not self.controller.publish_is_running:
```

Comment:
Does this same check maybe need to be done for CTRL+S?

Reply:
Not this, but different.